Rajitha/koding-practise-1.js

```diff
@@ -0,0 +1,157 @@
+/*********************************************************************
+* Without using any built in functions, write a function to filter an
+* array for unique values
+* Input: [1,1,2,3,5,7,7,9]
+* Output: [1,2,3,5,7,9]
+**********************************************************************/
+// Using For loop
+const getUniqueArray = (arr) => {
+  let newArray = [];
+  let dupArray = [];
+  for (let i = 0; i <= arr.length; i++){
+    for (let j = i + 1; j <= arr.length; j++) {
+      if (arr[i] == arr[j]) {
```

Comment:
This is good for the given case but what happens here if there is more than 2 of a specific element in a row in the input array?